tests/storage/restore.go

```diff
@@ -1752,26 +1756,38 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 					)
 
 					return libvmi.NewVirtualMachine(
-						libstorage.RenderVMIWithDataVolume(dataVolume.Name, sourceDV.Namespace, libvmi.WithCloudInitNoCloud(libvmifact.WithDummyCloudForFastBoot())),
+						libstorage.RenderVMIWithDataVolume(dataVolume.Name, testsuite.NamespaceTestDefault, libvmi.WithCloudInitNoCloud(libvmifact.WithDummyCloudForFastBoot())),
```

Comment:
> Also fixed a bug in the test where the VM was created in the source
namespace instead of a different namespace (the test checks for
cross namespace).

This is it, right? very nice catch :pray: 
Maybe `testsuite.GetTestNamespace(nil)` fits better logically here?

BTW, I traced it to https://github.com/kubevirt/kubevirt/commit/20fc6d8a10d15f47bbe5c6ed85d275b41bf733e4#diff-a9f8dfbfa9d6d766bcefa6f3d91bae717c88c3590685409587e304511f9c09d8L1782-R1770
which as far as I can tell did not make any release branch

Reply:
Thanks, yeah it was merged not too long ago. I will use your suggestion